app/scripts/views/components/virtualStudy/virtualStudy.js

```diff
@@ -0,0 +1,353 @@
+'use strict';
+(function(Vue, $, vcSession, iViz) {
+  Vue.component('virtualStudy', {
+    template:
+    '<div class="virtual-study">' +
+    '<div class="virtual-study-btn">' +
+    'Virtual Study</div></div>',
+    props: {
+      selectedSamplesNum: {
+        type: Number,
+        default: 0
+      },
+      stats: {
+        type: Object
+      },
+      updateStats: {
+        type: Boolean,
+        default: false
+      },
+      createdQtip: {
+        type: Boolean,
+        default: false
+      }
+    },
+    data: function() {
+      return {
+        savedVS: null,
+        virtualStudyName: 'Selected Study',
+        virtualStudyDescription: ''
+      };
+    },
+    methods: {
+      styleDisplay: function(el, action) {
+        el.css('display', action);
+      },
+      showDialog: function(tooltip) {
```

Comment:
I would prefer to go with one function and pass more parameters `className` and `display`

Reply:
I actually prefer the current implementation. The logic is clear. We don't need to remember where are places using this function and change the class name individually if that happens